testing/install_external_dependencies.sh

```diff
@@ -35,15 +35,28 @@ fi
 
 # Install Python dependencies for Python specific tests
 if [[ -n "$PYTHON" ]] ; then
-  wget https://repo.continuum.io/miniconda/Miniconda${PYTHON}-4.2.12-Linux-x86_64.sh -O miniconda.sh
-  bash miniconda.sh -b -p $HOME/miniconda
+  case "$(uname)" in
+  Darwin)
+    os=MacOSX
+    ;;
+  *)
+    os=Linux
+    ;;
+  esac
+  if [ ! -e miniconda.sh ]; then
+    wget "https://repo.continuum.io/miniconda/Miniconda${PYTHON}-4.2.12-$os-$(uname -m).sh" -O miniconda.sh.part
+    mv miniconda.sh.part miniconda.sh
```

Comment:
why this?

Reply:
In case download is Ctrl+C-ed and partial file remains.